great_expectations/expectations/expectation.py

```diff
@@ -1543,7 +1545,7 @@ def _validate(
         return _format_map_output(
             result_format=parse_result_format(result_format),
             success=success,
-            element_count=filtered_row_count,
+            element_count=total_count,
             nonnull_count=filtered_row_count,
```

Comment:
Given the null vs. filtered distinction that's being pointed out here, I believe it would be more appropriate to actually add a new parameter called "filtered_row_count" into the result format, rather than call these "null values"

Reply:
Since this is a new expectation type, let's just directly add that to _format_map_output as an option.

Reply:
@jcampbell I agree with this idea -- how about I make a JIRA item for it and one of the pod team members will address in a follow-on PR?  Thanks!

Reply:
👍

Reply:
This is now captured in JIRA: `https://superconductive.atlassian.net/browse/GREAT-92`.  Thanks.